user-interface/src/App.tsx

```diff
@@ -14,8 +14,8 @@ function App() {
         <HeaderNavBar />
         <div className="body">
           <Routes>
-            <Route path="/cases" element={<CaseList />}></Route>
             <Route path="/" element={<Home />}></Route>
+            <Route path="/cases" element={<CaseList />}></Route>
             <Route path="*" element={<NotFound />}></Route>
```

Comment:
Default screen should be "/".  I don't know if defining "/cases" route first would affect the routing but I believe it handles the path expressions in a top->down manor.